### PR TITLE
added something in branch

### PR DESCRIPTION
**Pull request by simonmarqvard**
trying to do something smart

### CODE REVIEW

**Review by julietarod — COMMENTED** [review]
I don't like these changes

File: index.html [hunk 1]
```diff
@@ -8,6 +8,19 @@
 </head>
 <body>
     <div> simons div </div>
```
Comment (hinausman) [inline index.html:10]: test comment

**Comment by camilaabreu17** [comment]
all good

File: index.html (hunk 1 above)
Comment (Deepalijain126) [reply index.html:10]: hiiii

**Review by aletilalitha — COMMENTED** [review]
review the code

**Review by Aditi-Sawardekar — COMMENTED** [review]
There are too many divs

**Review by Deepalijain126 — COMMENTED** [review]
hello good work

**Pull request merged**